app/adapters/beta-feature.js

```diff
@@ -0,0 +1,35 @@
+import Ember from 'ember';
+import V3Adapter from 'travis/adapters/v3';
+
+const { service } = Ember.inject;
+
+export default V3Adapter.extend({
+  auth: service(),
+
+  buildURL(modelName, id, snapshot, requestType) {
+    let url = this._super(...arguments);
+    if (requestType === 'GET') {
+      let replacement = `user/${this.get('auth.currentUser.id')}/beta_features`;
+      url = url.replace(/beta_features/, replacement);
+    } else {
+      if (requestType === 'PUT') { requestType = 'PATCH'; }
+      let replacement = `user/${this.get('auth.currentUser.id')}/beta_feature`;
```

Comment:
Right. I'm sending `id`, it's just not apparent from this code as it's automatically appended.